jwst/extract_2d/grisms.py

```diff
@@ -445,20 +446,24 @@ def extract_grism_objects(input_model,
                                                  xcenter_model &
                                                  ycenter_model &
                                                  order_model) | tr
-
-                ext_data = input_model.data[ymin: ymax + 1, xmin: xmax + 1].copy()
-                ext_err = input_model.err[ymin: ymax + 1, xmin: xmax + 1].copy()
-                ext_dq = input_model.dq[ymin: ymax + 1, xmin: xmax + 1].copy()
+                from gwcs.utils import _toindex
+                y_slice = slice(_toindex(ymin), _toindex(ymax))
+                x_slice = slice(_toindex(xmin), _toindex(xmax))
+                ext_data = input_model.data[y_slice, x_slice].copy()
+                # ext_err = input_model.err[ymin: ymax + 1, xmin: xmax + 1].copy()
```

Comment:
Is there a reason to leave the commented lines in here?

Reply:
no